src/bin/fish.rs

```diff
@@ -440,61 +302,6 @@ fn fish_parse_opt(args: &mut [WString], opts: &mut FishCmdOpts) -> ControlFlow<i
             'f' => opts.features = w.woptarg.unwrap().to_owned(),
             'h' => opts.batch_cmds.push("__fish_print_help fish".into()),
             'i' => opts.is_interactive_session = true,
-            'I' => {
```

Comment:
Need to remove this from the optspecs as well ("fish --install" is crashing with unexpected retval from WGetopter)